CHANGELOG.rst

```diff
@@ -16,6 +16,11 @@ Unreleased
 * Add script to get github action errors
 * Add script to republish failed events
 
+[2.1.0] - 2023-06-01
+~~~~~~~~~~~~~~~~~~~~
+
+* Adds test factory
```

Comment:
Two high-level comments:
1. This functionality seems low on docs about the new functionality (README or longer docstrings, etc.), including this changelog entry.
2. I don't think this repo is the right place for this, but we should discuss. See https://github.com/edx/edx-arch-experiments/blob/main/docs/decisions/0001-purpose-of-this-repo.rst. I'm not sure that this aligns. That said, the missing docs makes it hard for me to be sure.

Reply:
I agree. I think this might make more sense in devstack

Reply:
> I agree. I think this might make more sense in devstack

Do we have any history of the decision to use edx-django-utils that we can provide here or are on the new PR?